pynucastro/rates/rate.py

```diff
@@ -647,6 +647,18 @@ def eval(self, T, *, rho=None, comp=None):
 
         raise NotImplementedError("base Rate class does not know how to eval()")
 
+    def function_string_py(self):
+        """Return a string containing the python function that
+        computes the rate.
+
+        Raises
+        ------
+        NotImplementedError
+
+        """
+
+        raise NotImplementedError("base Rate class does not know how to eval()")
```

Comment:
do we want a different error message here?

Reply:
yes.  Good catch.  Fixed